org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java

```diff
@@ -27,6 +29,7 @@
 
 import aQute.lib.io.ByteBufferOutputStream;
 import aQute.lib.io.IO;
+import aQute.libg.tuple.Pair;
```

Comment:
I would prefer to not increase dependency on bnd packages. So please don't use this. Not using this will probably remove your need to add bndlib as a dependency.

Reply:
Makes sense, @bjhargrave! Replaced use of `aQute.libg.tuple.Pair` with `java.util.AbstractMap.SimpleEntry`

Reply:
Neither `aQute.libg.tuple.Pair` nor `java.util.AbstractMap.SimpleEntry` are required with the latest 2-pass approach